SDK Showcase/SDK Showcase/Interactors/SDKInteractor.swift

```diff
@@ -142,6 +160,14 @@ extension SDKInteractorReal: ChangePinDelegate {
     func userClient(_ userClient: any UserClient, didFailToChangePinForUser profile: any UserProfile, error: any Error) {
         appState.setSystemError(string: error.localizedDescription)
     }
+    
+    func fetchUserProfiles() {
```

Comment:
this is the wrong place for this piece of the code - it's not related to changing PIN